PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -134,7 +134,10 @@
 run3_common.toModify(MicroEventContent, outputCommands = MicroEventContent.outputCommands + _run3_common_extraCommands)
 # --- 
 
-_pp_on_AA_extraCommands = ['keep *_packedCandidateMuonID_*_*']
+_pp_on_AA_extraCommands = [
+	'keep *_packedCandidateMuonID_*_*',
+	'keep floatedmValueMap_*TrackChi2_*_*'
```

Comment:
wildcards should be avoided for (reco-related) products. Please spell out the producing module name in full